webpack-intro/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "webpack-intro",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "lindsey",
+  "license": "ISC",
+  "dependencies": {
+    "jquery": "^3.6.0"
+  },
+  "devDependencies": {
+    "webpack-cli": "^4.9.2"
```

Comment:
`webpack` should be listed under `devDependencies` here with `webpack-cli`

Reply:
Thank you! I wasn't too clear on whether or not the install was just the cli or both, must have just been the cli!